libraries/chain/database.cpp

```diff
@@ -2259,6 +2260,47 @@ namespace golos { namespace chain {
             }
         }
 
+        uint64_t database::pay_delegators(const account_object& delegatee, const comment_vote_object& cvo, uint64_t claim) {
+            uint64_t delegators_reward = 0;
+            const auto& vdo_idx = get_index<vesting_delegation_index>().indices().get<by_received>();
+            const auto& vdeo_idx = get_index<vesting_delegation_expiration_index>().indices().get<by_received>();
+            for (auto& dvir : cvo.delegator_vote_interest_rates) {
+                auto delegator_claim = claim * dvir.interest_rate / STEEMIT_100_PERCENT;
+
+                if (delegator_claim == 0) {
+                    continue;
+                }
+
+                delegators_reward += delegator_claim;
+
+                const auto& delegator = get_account(dvir.account);
+                asset delegator_vesting = create_vesting(delegator, asset(delegator_claim, STEEM_SYMBOL));
+                if (dvir.payout_strategy == to_delegated_vesting) {
+                    modify(delegator, [&](account_object& a) {
+                        a.delegated_vesting_shares += delegator_vesting;
```

Comment:
And looks like - if no records in vdo_idx and vdeo_idx we can skip adding of  delegated shares to account.